qemu/tests/win_heavyload.py

```diff
@@ -139,9 +139,18 @@ def add_option(cmd, key, val):
             if not installed and download_url:
                 utils.system("rm -f %s/HeavyLoad*.exe" % tmp_dir)
                 session.cmd("del /f /s %sHeavyLoad*.exe" % dst)
-            if loop_session_cmd(session, check_running_cmd):
-                if not loop_session_cmd(session, stop_cmd):
-                    raise error.TestFail("Unable to terminate heavyload "
-                                         "process")
+            i = 0
```

Comment:
it is used not only here,  so can not be removed.
ex:
`        if not loop_session_cmd(session, check_running_cmd):                                                              
            raise error.TestError("heavyload process is not started")  `